.gitignore

```diff
@@ -55,6 +55,3 @@ Thumbs.db
 
 # Generated JSON files
 *_gen.json
-
-# Discord config file
-apps/somnia/src/config.json
```

Comment:
Shouldn't you add the new path here?

Reply:
Also, can you upload a blank `config.empty.json` or similar with the needed fields